unitTests/testTensorOpsTwoSizes.hpp

```diff
@@ -669,6 +669,65 @@ class TwoSizesTest : public ::testing::Test
         } );
   }
 
+  void testScaledAdd()
+  {
+    ArrayViewT< T, 3, 2 > const matrixA_IJK = m_matrixA_IJK.toView();
+    ArrayViewT< T, 3, 1 > const matrixA_IKJ = m_matrixA_IKJ.toView();
+    ArrayViewT< T, 3, 0 > const matrixA_KJI = m_matrixA_KJI.toView();
+
+    ArrayViewT< T const, 3, 2 > const matrixB_IJK_view = m_matrixB_IJK.toView();
+    ArrayViewT< T const, 3, 1 > const matrixB_IKJ_view = m_matrixB_IKJ.toView();
+    ArrayViewT< T const, 3, 0 > const matrixB_KJI_view = m_matrixB_KJI.toView();
+
+    T const ( &matrixB_local )[ N ][ M ] = m_matrixB_local;
+
+    std::ptrdiff_t const matrixSeed = m_matrixASeed;
+
+    forall< POLICY >( 1, [matrixA_IJK, matrixA_IKJ, matrixA_KJI, matrixB_IJK_view, matrixB_IKJ_view, matrixB_KJI_view, matrixB_local, matrixSeed ]
+                      LVARRAY_HOST_DEVICE ( int )
+        {
+          #define _TEST( dstMatrix, srcMatrix ) \
+            fill( dstMatrix, matrixSeed ); \
+            tensorOps::scaledAdd< N, M >( dstMatrix, srcMatrix, scale ); \
+            CHECK_EQUALITY_2D( N, M, dstMatrix, result ); \
```

Comment:
@corbett5 the vector version of this function uses a tolerance `CHECK_NEAR_1D`. Is this going to be ok? `CHECK_NEAR_2D` does not seem to exist.

Reply:
Ideally they would all use `CHECK_EQUALITY_*`, but FMA gives different answers with some compilers and devices. Can you try it out on Lassen with both clang and GCC in release and debug? If that works then it's fine with me, otherwise adding a `CHECK_NEAR_2D` should be pretty easy.